nntrainer/src/neuralnet.cpp

```diff
@@ -431,15 +486,18 @@ int NeuralNetwork::init() {
       NN_INI_RETURN_STATUS();
       break;
     }
-    const char *acti_str = iniparser_getstring(
-      ini, (layer_name + ":Activation").c_str(), unknown);
-    ActiType act = (ActiType)parseType(acti_str, TOKEN_ACTI);
+    previous_dim = layers.back()->getOutputDimension();
 
-    layers.back()->setActivation(act);
-    status = initActivationLayer(act);
-    NN_INI_RETURN_STATUS();
+    if (layer_type != LAYER_IN && layer_type != LAYER_POOLING2D &&
```

Comment:
Do we need this condition here? 
IMO, the activation layer should be able to add at the end of any layer as our implementation allows it.
This might add `ACT_NONE` in some of the cases which we can optimize out later.

Reply:
After rebase, this can be removed.